tests/foundry/common/documents/actor.test-d.ts

```diff
@@ -101,9 +103,8 @@ declare namespace BoilerplateActorBase {
 
 class BoilerplateActorBase<
   Schema extends BoilerplateActorBase.Schema = BoilerplateActorBase.Schema,
-  BaseData extends Record<string, any> = Record<never, never>,
   DerivedData extends Record<string, any> = Record<never, never>,
-> extends foundry.abstract.TypeDataModel<Schema, Actor.ConfiguredInstance, BaseData, DerivedData> {
+> extends foundry.abstract.TypeDataModel<Schema, Actor.ConfiguredInstance, DerivedData> {
```

Comment:
This change I made and forgot to revert is wrong, even if it's just some tests, leaving it owuld probably cause confusions in the future.